### ISSUE TITLE
DPLA image search not working for any shortcodes

### ISSUE THREAD

**Issue by mmwoods2019** [NEU-DSG/toolkit_requests#117]
When inserting a shortcode, I've tried searching for DPLA images on two sites (Toolkit Test site and  CERES Exhibit Toolkit) and no results showed up in either case. I tried all of the shortcodes and searched various terms, but it made no difference. Megan Barney also went on the ECDA site and tried searching DPLA images with every display option for the shortcode and no results showed up for her as well.

**Comment by patrickmj** [comment]
Two quick debugging questions:

1) Is this happening across various browsers? (It might be an issue in the javascript, so different browsers could hypothetically behave differently -- even across versions of browsers. That's kinda a long-shot, but worth checking).

2) If you recall, when is the last time you successfully added a DPLA image? If it's been a while, we might have to look back through the code to just before that point in time. 

3) If you remember a known DPLA id that used to work, could you post it here? And, maybe a link to the CERES site/page where it is (or should be!) visible?

I've had the same problem with search on my local installation, so it doesn't look like a server thing right now. But an id to something that should work will help move things forward.

Thanks much!

**Comment by mmwoods2019** [comment]
Thanks for your response/suggestions, Patrick! Here are my notes on the questions:

1. I've tried this on both Chrome (Version 69.0.3497.100) and Safari (Version 12.0) and the same thing happens when I search for a DPLA image while inserting a shortcode. So it looks like the browser may not be the issue. This is what happens when I search for an item:

<img width="1343" alt="screen shot 2018-10-12 at 9 44 49 am" src="https://user-images.githubusercontent.com/43454418/46873350-48b7db80-ce04-11e8-9d3d-cad575d6bbbf.png">

2. The last time I specifically looked at DPLA images with shortcodes was just over two weeks ago when I looked into another issue with DPLA images [Issue #115 ]. In that case, searching DPLA images didn't work with the Timeline short code but it was working for all of the other shortcodes. 
 
3. I used DPLA images when creating a Practice page on the Toolkit Test site: https://toolkittraining.dsg.northeastern.edu/practice-page/. It appears that DPLA images are not even showing up and just have little boxes with question marks in the center of them. Here are the DPLA ids: "dpla:6e1eb5f9532d92816e1fdc5d01405ea9,dpla:11f143e718862017891c876831d0ba4d,dpla:81d99397747eb271dc8487b4621d5cff"

<img width="1248" alt="screen shot 2018-10-12 at 9 59 45 am" src="https://user-images.githubusercontent.com/43454418/46873828-9a149a80-ce05-11e8-85cf-bd278a3cf386.png">

I should also note that this is for sites using both classic editor and Gutenberg.
